clang/docs/UsersManual.rst

```diff
@@ -2660,7 +2660,7 @@ usual build cycle when using sample profilers for optimization:
 
      > clang-cl /O2 -gdwarf -gline-tables-only ^
        /clang:-fdebug-info-for-profiling /clang:-funique-internal-linkage-names ^
-       /fprofile-sample-use=code.prof code.cc /Fe:code /fuse-ld=lld /link /debug:dwarf
```

Comment:
I take it this wasn't working? (Neither `/fprofile-sample-use` nor `/fuse-ld`?)

Reply:
Yes. I checked options.td.
-fprofile-sample-use was not a CLOption and there's no /fuse-ld.